src/frontends/tensorflow/src/tf_utils.cpp

```diff
@@ -463,12 +464,24 @@ bool propagate_conditional_flow(const OutputVector& ov_inputs,
     return to_propagate;
 }
 
+namespace {
+shared_ptr<op::v0::Parameter> get_parent_parameter(const shared_ptr<op::v0::Result>& node) {
+    const auto input_values = node->input_values();
+    if (input_values.empty())
+        return {};
+    return as_type_ptr<v0::Parameter>(input_values[0].get_node_shared_ptr());
+}
+bool has_parent_parameter(const shared_ptr<op::v0::Result>& node) {
+    return get_parent_parameter(node) != nullptr;
+}
+}  // namespace
+
 // create Loop operation corresponding to TensorFlow While operation
-shared_ptr<v5::Loop> create_loop_for_tf_while(const std::string& while_node_name,
-                                              const shared_ptr<Model>& body_model,
-                                              const shared_ptr<Model>& cond_model,
-                                              const OutputVector& ov_inputs,
-                                              const shared_ptr<Model>& prior_cond_model) {
+ov::OutputVector create_loop_for_tf_while(const std::string& while_node_name,
```

Comment:
let us do not set tensor names in this routine. It will just create necessary sub-graph